airflow/migrations/utils.py

```diff
@@ -35,7 +35,8 @@ def get_mssql_table_constraints(conn, table_name):
      FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS AS tc
      JOIN INFORMATION_SCHEMA.CONSTRAINT_COLUMN_USAGE AS ccu ON ccu.CONSTRAINT_NAME = tc.CONSTRAINT_NAME
      WHERE tc.TABLE_NAME = '{table_name}' AND
```

Comment:
😱  This should be a placeholder/param pass to execute, not built in to the string. (Separate to this PR)